test/text/web_vtt_generator_unit.js

```diff
@@ -22,9 +24,9 @@ describe('WebVttGenerator', () => {
           shakaCue2,
         ],
         'WEBVTT\n\n' +
-        '00:00:20.000 --> 00:00:40.000\n' +
+        '00:00:20.000 --> 00:00:40.000 align:left vertical:lr\n' +
         'Test\n\n' +
-        '00:00:40.000 --> 00:00:50.000\n' +
+        '00:00:40.000 --> 00:00:50.000 align:middle\n' +
```

Comment:
Could you instead create a separate test case for these?

Reply:
The default value is center (https://github.com/google/shaka-player/blob/master/lib/text/cue.js#L83), do you consider that it's necesaary separate the test?

Reply:
Sorry.  I wasn't referring to align:middle specifically, but all of the settings you've added support for in this PR.

Reply:
Done!